test/manual/definitions/ohi/linux/nginx-mysql-jmx-redis-linux2.json

```diff
@@ -1,44 +0,0 @@
-{
-  "global_tags": {
-    "owning_team": "OpenSource",
-    "Environment": "development",
-    "Department": "Product",
-    "Product": "Virtuoso"
-  },
-
-  "resources": [{
-    "id": "nmjrlinux2",
-    "display_name": "NginxMySqlJmxRedisLinux2Host",
-    "provider": "aws",
-    "type": "ec2",
-    "size": "t3.small",
-    "ami_name": "amazonlinux-2-base*",
-    "user_name": "ec2-user"
-  }],
-
-  "services": [{
-    "id": "nginx1",
-    "destinations": ["nmjrlinux2"],
-    "source_repository": "https://github.com/newrelic/open-install-library.git",
-    "deploy_script_path": "test/deploy/linux/nginx/install/linux2/roles",
-    "port": 80
-  },{
-    "id": "mysql1",
-    "destinations": ["nmjrlinux2"],
-    "source_repository": "https://github.com/newrelic/open-install-library.git",
-    "deploy_script_path": "test/deploy/linux/mysql/install/rhel/roles",
-    "port": 9999
-  },{
-    "id": "jmx1",
```

Comment:
@noahmmcgivern just remove the "jmx1" service definition and keep the rest of the test file, you might rename without the `-jmx` in the filename.